ansible/group_vars/all/defaults.yaml

```diff
@@ -3,9 +3,9 @@ python_path: ""
 
 # This is also in images/common.yaml as that's where the go code expects it to be.
 # If it's not there, the kubernetes_full_version will have "None" for a version number.
-kubernetes_version: "1.24.6"
+kubernetes_version: "1.25.4"
 
-containerd_version: "1.4.13"
+containerd_version: "1.6.10"
```

Comment:
I wonder if we also need to update the Containerd config.

Reply:
Not just yet.

We already use version 2 of the config.

We do use the deprecated registry configuration, and I've filed https://d2iq.atlassian.net/browse/D2IQ-94431 to address that in the future.